### ISSUE TITLE
3d prints do not support unicode strings in label names

### ISSUE THREAD

**Issue by Allymonies** [SwitchCraftCC/sc-peripherals#11]
When printing a .3dj file with a unicode string in the label, the resulting label appears mangled, rather than displaying the symbols specified, necessitating the use of anvils to put symbols in the names of 3d prints.

Test file:
```
{
  "label": "\u2661 Trans Rights \u2661",
  "collideWhenOff": true,
  "collideWhenOn": true,
  "lightLevel": 15,
  "redstoneLevel": 0,
  "shapesOff": [
    {"bounds":[0,11,15,16,13,16],"tint":"FFFFFF","texture":"minecraft:block/light_blue_wool"},
    {"bounds":[0,9,15,16,11,16],"tint":"FFFFFF","texture":"minecraft:block/pink_wool"},
    {"bounds":[0,7,15,16,9,16],"tint":"FFFFFF","texture":"minecraft:block/white_wool"},
    {"bounds":[0,5,15,16,7,16],"tint":"FFFFFF","texture":"minecraft:block/pink_wool"},
    {"bounds":[0,3,15,16,5,16],"tint":"FFFFFF","texture":"minecraft:block/light_blue_wool"}
  ],
  "shapesOn": []
}
```

![khplw 1](https://user-images.githubusercontent.com/7727003/214221192-92e9864c-d8c1-40b2-b0ca-fa34887917c5.png)

**Comment by Lemmmy** [comment]
Frankly I'm not optimistic about fixing this one, I feel like it's going to cause more trouble further down the line, but we'll see how it goes

**Comment by SquidDev** [comment]
I'd recommend using `StandardCharsets.UTF_8.decode(buffer).toString()` instead of depending on Cobalt's behaviour.

**Comment by Lemmmy** [comment]
fixed in e38b194
